Lib/zipfile/__init__.py

```diff
@@ -439,7 +439,7 @@ def __repr__(self):
         result.append('>')
         return ''.join(result)
 
-    def FileHeader(self, zip64=None):
+    def FileHeader(self, zip64):
```

Comment:
Sorry, I figured it was ok to change it since it's not really documented anywhere and seems like a very internal-only method, despite the lack of an `_`-prefix. I've added back the default (but changed it to `False` since it doesn't dynamically pick if it uses zip64 extension or not anymore.